[2021-09-2820:24:21] [ ERROR] [GlobalExceptionHandler:18] Paciente no se ha encontrado en la base de datos
[2021-09-2820:24:31] [ ERROR] [GlobalExceptionHandler:18] Ondontologo no se encuentra en la base de datos
[2021-09-2820:24:45] [ ERROR] [GlobalExceptionHandler:18] Turno no se encuentra en la base de datos
[2021-09-2820:26:14] [ ERROR] [GlobalExceptionHandler:18] Domicilio no se ha encontrado en la base de datos
[2021-09-2921:22:47] [ ERROR] [GlobalExceptionHandler:18] Domicilio no se ha encontrado en la base de datos
[2021-09-2921:23:44] [ ERROR] [GlobalExceptionHandler:18] Paciente no se ha encontrado en la base de datos
[2021-09-2921:24:30] [ ERROR] [GlobalExceptionHandler:18] Ondontologo no se encuentra en la base de datos
[2021-09-2921:25:07] [ ERROR] [GlobalExceptionHandler:18] Already had POJO for id (java.lang.Integer) [[ObjectId: key=4, type=com.fasterxml.jackson.annotation.ObjectIdGenerators$IntSequenceGenerator, scope=java.lang.Object]] (through reference chain: com.example.tb_integrador_lindaRamirez.controller.DTO.TurnoDTO["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[0]->com.example.tb_integrador_lindaRamirez.entity.Turno["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[1]->com.example.tb_integrador_lindaRamirez.entity.Turno["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[2]->com.example.tb_integrador_lindaRamirez.entity.Turno["id"])
[2021-09-2921:25:18] [ ERROR] [GlobalExceptionHandler:18] Already had POJO for id (java.lang.Integer) [[ObjectId: key=3, type=com.fasterxml.jackson.annotation.ObjectIdGenerators$IntSequenceGenerator, scope=java.lang.Object]] (through reference chain: com.example.tb_integrador_lindaRamirez.controller.DTO.TurnoDTO["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[0]->com.example.tb_integrador_lindaRamirez.entity.Turno["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[1]->com.example.tb_integrador_lindaRamirez.entity.Turno["id"])
[2021-09-2921:25:25] [ ERROR] [GlobalExceptionHandler:18] Already had POJO for id (java.lang.Integer) [[ObjectId: key=3, type=com.fasterxml.jackson.annotation.ObjectIdGenerators$IntSequenceGenerator, scope=java.lang.Object]] (through reference chain: com.example.tb_integrador_lindaRamirez.controller.DTO.TurnoDTO["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[0]->com.example.tb_integrador_lindaRamirez.entity.Turno["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[1]->com.example.tb_integrador_lindaRamirez.entity.Turno["id"])
[2021-09-2921:25:42] [ ERROR] [GlobalExceptionHandler:18] Already had POJO for id (java.lang.Integer) [[ObjectId: key=3, type=com.fasterxml.jackson.annotation.ObjectIdGenerators$IntSequenceGenerator, scope=java.lang.Object]] (through reference chain: com.example.tb_integrador_lindaRamirez.controller.DTO.TurnoDTO["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[0]->com.example.tb_integrador_lindaRamirez.entity.Turno["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[1]->com.example.tb_integrador_lindaRamirez.entity.Turno["id"])
[2021-09-2921:26:04] [ ERROR] [GlobalExceptionHandler:18] Turno no se encuentra en la base de datos
[2021-09-2921:26:13] [ ERROR] [GlobalExceptionHandler:18] Already had POJO for id (java.lang.Integer) [[ObjectId: key=2, type=com.fasterxml.jackson.annotation.ObjectIdGenerators$IntSequenceGenerator, scope=java.lang.Object]] (through reference chain: com.example.tb_integrador_lindaRamirez.controller.DTO.TurnoDTO["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[0]->com.example.tb_integrador_lindaRamirez.entity.Turno["odontologo"]->com.example.tb_integrador_lindaRamirez.entity.Odontologo["turnos"]->java.util.HashSet[0]->com.example.tb_integrador_lindaRamirez.entity.Turno["id"])
[2021-09-2922:58:29] [ ERROR] [GlobalExceptionHandler:18] Domicilio no se ha encontrado en la base de datos
[2021-09-2922:58:54] [ ERROR] [GlobalExceptionHandler:18] Paciente no se ha encontrado en la base de datos
[2021-09-2923:00:13] [ ERROR] [GlobalExceptionHandler:18] Ondontologo no se encuentra en la base de datos
[2021-09-2923:03:18] [ ERROR] [GlobalExceptionHandler:18] Request method 'POST' not supported
